internal/rty/layouts_test.go

```diff
@@ -133,3 +134,39 @@ func TestMinWidthInNestedConcatLayoutOverflow(t *testing.T) {
 	i := NewInteractiveTester(t, screen)
 	i.Run("min width concat text overflow", 8, 1, f2)
 }
+
+func TestTailLayout(t *testing.T) {
+	sc := tcell.NewSimulationScreen("")
+	err := sc.Init()
+	assert.NoError(t, err)
+	sc.SetSize(8, 1)
```

Comment:
Is it possible these four lines were cargo culted and unnecessary? I wasn't able to tell in a few minutes what purpose they're serving here (and even elsewhere I'm not sure if there's a good reason to do this instead of InteractiveTester)

Reply:
ya, i think this is vestigal from before InteractiveTester set the screen size, removed!